tests/process_monitor/test_process_monitor.py

```diff
@@ -493,10 +493,41 @@ def test_MantarrayProcessesMonitor__logs_errors_from_subprocesses(
 def test_MantarrayProcessesMonitor__hard_stops_and_joins_processes_and_logs_queue_items_when_error_is_raised_in_ok_comm_subprocess(
     mocker, test_process_manager_creator, test_monitor
 ):
-    expected_ok_comm_item = f"ok_comm_queue_item, {str(bytes(4))}"
-    expected_file_writer_item = "file_writer_queue_item"
-    expected_da_item = "data_analyzer_queue_item"
-    expected_server_item = "server_manager_queue_item"
+
+    expected_ok_comm_item = {
+        "board_0": {"file_writer_to_data_analyzer": [], "outgoing_data": []},
```

Comment:
I mentioned this too already, okcomm does not have have a file writer to data analyzer queue, only file writer and data analyzer have that queue